libraries/botbuilder-dialogs-adaptive-testing/src/testScript.ts

```diff
@@ -58,6 +63,10 @@ export class TestScript {
         bot.userState = new UserState(new MemoryStorage());
         ResourceExtensions.useResourceExplorer(bot, resourceExplorer);
         LanguageGeneratorExtensions.useLanguageGeneration(bot);
+        
+        for (let i = 0; i < this.userTokenMocks.length; i++) {
```

Comment:
`forEach` makes this a bit simpler: `this.userTokenMocks.forEach(mock => mock.setup(testAdapter));`